### ISSUE TITLE
let's add "QA reviews"

### ISSUE THREAD

**Issue by yegor256** [zerocracy/farm#73]
Let's pay for a job only after QA review is completed. Let's pay 15 minutes for each QA review, no matter what is the result of it (either good or bad). More: http://at.teamed.io/policy.html#24

**Comment by 0crat** [comment]
The user @caarlos0[/z](https://www.0crat.com/u/caarlos0) resigned from [#73](https://github.com/zerocracy/farm/issues/73), please stop working

**Comment by 0crat** [comment]
The job [#73](https://github.com/zerocracy/farm/issues/73) assigned to @neonailol[/z](https://www.0crat.com/u/neonailol), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/410134025). The budget is 30 minutes, see [§4](http://datum.zerocracy.com/pages/policy.html#4). Please, read [§8](http://datum.zerocracy.com/pages/policy.html#8) and [§9](http://datum.zerocracy.com/pages/policy.html#9). If the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html).

**Comment by neonailol** [comment]
@0crat waiting

**Comment by 0crat** [comment]
> @0crat waiting ([here](https://github.com/zerocracy/farm/issues/73#issuecomment-366429747))

@neonailol The impediment for [#73](https://github.com/zerocracy/farm/issues/73) was registered successfully by @neonailol[/z](https://www.0crat.com/u/neonailol)

**Comment by neonailol** [comment]
@yegor256 it looks like this task is already implemented, am i wrong?

**Comment by yegor256** [comment]
@0crat refuse

**Comment by yegor256** [comment]
@neonailol you are right, it's done already.

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/zerocracy/farm/issues/73#issuecomment-368627197))

@yegor256 The user @neonailol[/z](https://www.0crat.com/u/neonailol) resigned from [#73](https://github.com/zerocracy/farm/issues/73), please stop working. Reason for job resignation: Order was cancelled

**Comment by 0crat** [comment]
The job [#73](https://github.com/zerocracy/farm/issues/73) is now out of scope
